app/models/delivery_option.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: delivery_options
+#
+#  id           :bigint           not null, primary key
+#  url          :string
```

Comment:
I'm trying to think of any other kinds of data we might need to store here. Possibly for mobile, would we want to have a deep link to directly open in an app?